docs/about/history.rst

```diff
@@ -29,6 +29,7 @@ Changed
    -  Add buttons to *Spline* inspector to quickly enable water features.
    -  Exposed control over *Spline* ribbon alignment - spline points now define the center of the ribbon by default.
    -  Caustics no longer render in shadows casted from objects underwater.
+   -  Added new CPU-based collision provider - *Baked FFT Data*.
```

Comment:
is this ok adding here? if you already made the new release this may need to be moved

Reply:
Ah yes sorry. Will have to be moved into 4.15 which I have setup in master.

Reply:
done